### PR TITLE
#726: Regression issue fix

### PR DESCRIPTION
**Pull request by dsuren1**
Fixes for #738

**Also look here:**
https://docs.google.com/document/d/12WjjeTAK8PqUZVK6vyB6a8B2C74LhBf1kyy26fRlQXo/edit

### CODE REVIEW

File: frontend/SiraNoMap.html [hunk 1]
```diff
@@ -10,12 +10,17 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
 <!--        <link rel="stylesheet" href="assets/application/conoscenze_ambientali/css/skin-mappa.css">-->
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://cesiumjs.org/releases/1.17/Build/Cesium/Widgets/widgets.css" />
```
Comment (mbarto) [inline frontend/SiraNoMap.html:13]: I don't think we should include Cesium related css and js, since we are not including the 3D view, could you revert these changes, please?

Comment (dsuren1) [reply frontend/SiraNoMap.html]: Sure

**Review by mbarto — APPROVED** [review]
